native-image-workshop/README.md

```diff
@@ -214,14 +231,14 @@ OK, so now we have added logging, let's see if it works by rebuilding and runnin
 
 ![User Input](./images/userinput.png)
 ```sh
-$ mvn clean package exec:exec -PJAVA
+$ mvn clean package exec:exec
 ```
 
 Great, that works. Now, let's build a native image using the maven profile:
```

Comment:
Great, that works. Now, let's build a native image using the **M**aven profile:

Reply:
updated